src/pages/Dashboard/RecentTransactions.tsx

```diff
@@ -1,7 +1,7 @@
-import { useSelector } from 'react-redux'
 import { Card } from 'components/Card/Card'
 import { Text } from 'components/Text'
 import { TransactionHistoryList } from 'components/TransactionHistory/TransactionHistoryList'
+import { useSelector } from 'react-redux'
```

Comment:
Revert this hunk, if you run `yarn lint --fix` you can see this doesn't pass lint and will get autofixed back to its original line